Travis CI build files

avinash-palleti opened the pull request:
Adding an autosync job file which keeps track of changes in upstream repos
and create an auto commit to update submodule SHA in repo.

Signed-off-by: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>

edersondisouza commented:
Looks interesting - but when/where is this thing run? In github itself? Some server that you control?

avinash-palleti commented:
It will run on Jenkins Slave node, cadence depends on what we decide.

edersondisouza commented:
Hmmm - I'm always wary of adding stuff that belongs to some external environment to repo. I mean, we use github and git, so things related to it are usually ok (like .gitignore). Imagine someone (TM) wanted to use avb-sh and also has a CI system, would this file be somehow in conflict with their own CI?

avinash-palleti commented:
Usually this is the way, people who use travis they add .travis.yaml file into repository. And these jenkins job files will not conflict with any other until someone try to keep same name for jobfile (Jenkinsfile or autosync.groovy).
We can have one hidden directory to store all these files ( .ci ).

edersondisouza commented:
Ok, go for it!

avinash-palleti committed:
Move ci infrastructure to travis from jenkins

Remove existing Jenkins job file and add travis-ci meta files.
This enables auto build for each PR using travis infrastructure.

Signed-off-by: Avinash Reddy Palleti <avinash.reddy.palleti@intel.com>

keerockl commented on travis.sh:
```diff
@@ -0,0 +1,30 @@
+cd $ROOT_DIR
+git clone git://git.alsa-project.org/alsa-lib.git
+cd alsa-lib
+git checkout v1.1.6
```
we support v1.1.7 now. can you update to v1.1.7?

keerockl replied:
Please update alsalib to v1.1.7

avinash-palleti replied:
Sure. Will submit new patch with an update.

avinash-palleti commented:
@edersondisouza @keerockl Can you please review and merge this.

edersondisouza reviewed (approved):
Looks ok.

avinash-palleti committed:
Update alsa-lib version

Update alsa-lib release version to v1.1.7
